src/ansys/geometry/core/designer/component.py

```diff
@@ -266,6 +266,11 @@ def name(self) -> str:
         """Name of the component."""
         return self._name
 
+    @name.setter
+    def name(self, value: str):
+        """Set the name of the component."""
+        self._name = value
+
```

Comment:
This information is not being send to the server. We need to be reflect the change on the server. Not just modify the name client side.